src/domain/networking/packets/EntityData.ts

```diff
@@ -9,31 +9,2063 @@
+    grabEquippable: boolean | undefined;
+    delegateToParent: boolean | undefined;
+    equippableLeftPositionOffset: vec3 | undefined;
+    equippableLeftRotationOffset: quat | undefined;
+    equippableRightPositionOffset: vec3 | undefined;
+    equippableRightRotationOffset: quat | undefined;
+    equippableIndicatorURL: string | undefined;
+    equippableIndicatorScale: vec3 | undefined;
+    equippableIndicatorOffset: vec3 | undefined;
+    density: number | undefined;
+    velocity: vec3 | undefined;
+    angularVelocity: vec3 | undefined;
+    gravity: vec3 | undefined;
+    acceleration: vec3 | undefined;
+    damping: number | undefined;
+    angularDampling: number | undefined;
+    restitution: number | undefined;
+    friction: number | undefined;
+    lifetime: number | undefined;
+    collisionless: boolean | undefined;
+    collisionMask: number | undefined;
+    dynamic: boolean | undefined;
+    collisionSoundURL: string | undefined;
+    actionData: ArrayBuffer | undefined;
+    cloneable: boolean | undefined;
+    cloneLifetime: number | undefined;
+    cloneLimit: number | undefined;
+    cloneDynamic: boolean | undefined;
+    cloneAvatarIdentity: boolean | undefined;
+    cloneOriginID: Uuid | undefined;
+    script: string | undefined;
+    scriptTimestamp: bigint | undefined;
+    serverScripts: string | undefined;
+    itemName: string | undefined;
+    itemDescription: string | undefined;
+    itemCategories: string | undefined;
+    itemArtist: string | undefined;
+    itemLicense: string | undefined;
+    limitedRun: number | undefined;
+    marketplaceID: string | undefined;
+    editionNumber: number | undefined;
+    entityInstanceNumber: number | undefined;
+    certificateID: string | undefined;
+    certificateType: string | undefined;
+    staticCertificateVersion: number | undefined;
+};
+
+type AnimationProperties = {
+    animationURL: string | undefined;
+    animationAllowTranslation: boolean | undefined;
+    animationFPS: number | undefined;
+    animationFrameIndex: number | undefined;
+    animationPlaying: boolean | undefined;
+    animationLoop: boolean | undefined;
+    animationFirstFrame: number | undefined;
+    animationLastFrame: number | undefined;
+    animationHold: boolean | undefined;
+};
+
+type ModelEntityProperties = {
+    shapeType: number | undefined;
+    compoundShapeURL: string | undefined;
+    color: vec3 | undefined;
+    textures: string | undefined;
+    modelURL: string | undefined;
+    modelScale: vec3 | undefined;
+    jointRotationsSet: boolean[] | undefined;
+    jointRotations: quat[] | undefined;
+    jointTranslationsSet: boolean[] | undefined;
+    jointTranslations: vec3[] | undefined;
+    groupCulled: boolean | undefined;
+    relayParentJoints: boolean | undefined;
+    blendShapeCoefficients: string | undefined;
+    useOriginalPivot: boolean | undefined;
+} & AnimationProperties;
+
+type EntityDataDetails = CommonEntityProperties & ModelEntityProperties;
```

Comment:
```suggestion
    blendShapeCoefficients: string | undefined;
    useOriginalPivot: boolean | undefined;
    animation: AnimationProperties | undefined;
};

type EntityDataDetails = CommonEntityProperties & ModelEntityProperties;
```
I was thinking have an `animation` property which is an object. This makes it the same as the "animation" property in the native scripting interface: https://apidocs.vircadia.dev/Entities.html#.EntityProperties-Model